js/ui/grid_core/ui.grid_core.focus.js

```diff
@@ -89,6 +128,10 @@ exports.FocusController = core.ViewController.inherit((function() {
                 rowsView = that.getView("rowsView"),
                 $tableElement;
 
+            if(!this._keyboardController._isVirtualScrolling()) {
```

Comment:
This condition is not necessery. For non virtual scrolling rowIndexOffset is zero.